### ISSUE TITLE
parameter-set heatpumpVisibility and it's usage

### ISSUE THREAD

**Issue by BlackEdder0815** [coolchip/luxtronik2#32]
Hi @coolchip,

I just checked your code because I want to create a PR with some additional parameters (de-aerate) - I currently have the issue, that at least one of my pumps get stuck every summer and needs to run regularly.
To my question: I found this mysterious byte-block 3005, what’s called “visibility”. It’s used for reading some values, like the “heating_curve_end_point”, which is checking the value “207”. The only “documentation” about this values I found here https://github.com/Bouni/Luxtronik-2/blob/master/documentation.md
But looking at 207, the related identifier of this is “ID_Visi_Sprachen_DANSK“ what doesn’t make any sense.
Can you elaborate, for what this visibility is used and what “207” stays for?
Happy to provide you a PR with some additional stuff.

The de-aerate props I have already in, just wondering, If I have anything to do with the visibility.

**Comment by coolchip** [comment]
Hello. The block with the visibility codes are only for visibility. The pump marks ununsed parameters with visibility=false. If you want a list with all possible codes, look here:
https://github.com/Bouni/Luxtronik-2/blob/master/documentation.md#meaning-of-the-data

**Comment by BlackEdder0815** [comment]
Thanks for the list.
According to this list, the “heating end-point” is only visible, when “Sprachen_DANSK” is set. This doesn’t make any sense to me… Or do I read/interpret this somehow wrong?

**Comment by coolchip** [comment]
You are right. That makes definitly no sense. I've no detailed information about those visibility parameters. If you are unsure, remove those if-clause with the visibility paramter. Instead of
`(heatpumpVisibility[167] === 1) ? heatpumpParameters[688] : 'no'`
use just:
`heatpumpParameters[688]`
There is nothing wrong with this.
